### PR TITLE
Update comments, refactor and comment nonexistent method call.

### PR DESCRIPTION
**Pull request by lucleads**
Updated test and format

### CODE REVIEW

**Comment by sstok** [comment]
You might wanna run php-cs-fixer because there a lot of unrelated CS changes.

Also, I'm not really sure what is the intend behind all these changes.

File: src/Extension/CoreExtension.php [hunk 1]
```diff
@@ -1216,41 +1217,42 @@ function twig_test_empty($value)
...
+     * @param bool $withContext
+     * @param bool $ignoreMissing Whether to ignore missing templates or not
+     * @param bool $sandboxed Whether to sandbox the template or not
+     *
+     * @return string The rendered template
+     * @throws LoaderError|SyntaxError
+     */
+function twig_include(Environment $env, array $context, $template, array $variables = [], bool $withContext = true, bool $ignoreMissing, bool $sandboxed): string
 {
-    $alreadySandboxed = false;
```
Comment (stof) [inline src/Extension/CoreExtension.php:1238]: The removal of this `$alreadySandboxed` changes the behavior of the method. Why ?

File: src/Extension/CoreExtension.php [hunk 2]
```diff
@@ -1216,41 +1217,42 @@ function twig_test_empty($value)
...
-        if (!$alreadySandboxed = $sandbox->isSandboxed()) {
+        if (!$sandbox->isSandboxed()) {
             $sandbox->enableSandbox();
         }
 
         foreach ((\is_array($template) ? $template : [$template]) as $name) {
             // if a Template instance is passed, it might have been instantiated outside of a sandbox, check security
             if ($name instanceof TemplateWrapper || $name instanceof Template) {
-                $name->unwrap()->checkSecurity();
+                $name->unwrap(); //TODO : Create checkSecurity() method
```
Comment (stof) [inline src/Extension/CoreExtension.php:1255]: `checkSecurity()` does exist in compiled template classes, as the SandboxExtension is adding it.. Your change broke the sandbox feature.

**Comment by stof** [comment]
To me, this PR seems to be doing only 2 things (unless I missed some stuff hidden in CS changes):

- changing coding standards in a bunch of places, which makes the codebase inconsistent (and there is no reason for that, as the coding standards  used by the project are something decided by the maintainers)
- breaking the sandbox feature

So I'm closing this PR.

**Pull request closed without merging**